storybook/cypress/e2e/dropdown-menu.cy.ts

```diff
@@ -118,7 +118,8 @@ describe("Dropdown menu - anchors", () => {
     cy.get("@menuitems").eq(1).should("have.focus");
   });
 
-  it("arrow up should cycle menu", { browser: "!firefox" }, () => {
+  // Todo: #2887
+  it.skip("arrow up should cycle menu", { browser: "!firefox" }, () => {
     cy.get("@button").focus().click();
```

Comment:
Deze skip is bewust in deze pr opgenomen?

Reply:
Run ging nu goed 👍